wpiutil/.styleguide

```diff
@@ -11,87 +11,75 @@ cppSrcFileInclude {
 }
 
 generatedFileExclude {
+  src/main/native/third_party/llvm/cpp/llvm/
```

Comment:
Seems like the redundant llvm (folder within cpp) is not required?  None of the other third_party directories have it.